index.css

```diff
@@ -222,10 +222,20 @@ h2{
 	text-align: center;
 }
 
+@media only screen and (max-width:1140px){
```

Comment:
Where did 1140 px come from? It's just another totally arbitrary number.

Reply:
So , I supposed to use media queries with em . Correct me if I am wrong.

Reply:
Yeah, since the page is position in em you have to use em to make it work properly.